client/js/render.js

```diff
@@ -90,6 +99,54 @@ function buildChatMessage(data) {
 	return msg;
 }
 
+function renderPreview(preview, msg) {
```

Comment:
Previews aren't appended in the template anymore.

I'm not scared of the `socket-events` folder lol. When refreshing the page, we're not getting the preview as a socket event, it doesn't make sense to load it from there. Everything gets loaded from the chain in `render` (build channels -> build messages -> etc.), and the rendering of previews is a part of that chain. So definitely makes sense to have it there.